server/cli/init/base-file-sources/hooks.js

```diff
@@ -1,44 +1,50 @@
 'use strict';
 
-/* eslint-disable no-var, prefer-arrow-callback */
+/* eslint-disable no-var*/
```

Comment:
Add this back and change back from arrows to function expressions, we can't guarantee this will be run in a version of Node that supports them. We could transpile this, but that's probably a whole different thing. otherwise it looks all good